.svglintrc.js

```diff
@@ -163,6 +164,136 @@ module.exports = {
+                    if (upperShorthandCurveCommand === command && x1Coord === xPrevCoord && y1Coord === yPrevCoord && x1Coord === x2Coord && y1Coord === y2Coord) {
+                      return true;
+                    }
+                    // Absolute bézier curve (C) having the same coordinate as the previous segment and last control point equal to the ending point
+                    if (upperCurveCommand === command && x1Coord === xPrevCoord && y1Coord === yPrevCoord && x2Coord === xCoord && y2Coord === yCoord) {
+                      return true;
+                    }
+                  }
+
+                  return (
+                    // Absolute horizontal direction (H) having the same x coordinate as the previous segment
+                    (upperHorDirectionCommand === command && x1Coord === xPrevCoord) ||
+                    // Absolute vertical direction (V) having the same y coordinate as the previous segment
+                    (upperVerDirectionCommand === command && x1Coord === yPrevCoord) ||
+                    // Absolute movement (M or L) having the same coordinate as the previous segment
+                    (upperMovementCommands.includes(command) && x1Coord === xPrevCoord && y1Coord === yPrevCoord)
+                  );
+                }
+              }
+            };
+            const invalidSegments = segments.filter(getInvalidSegments);
+
+            if (invalidSegments.length) {
+              invalidSegments.forEach(([command, x1Coord, y1Coord, ...rest]) => {
+                let readableSegment = `${command}${x1Coord}`;
+                if (y1Coord !== undefined) {
+                  readableSegment += ` ${y1Coord}`;
+                }
+                if (curveCommands.includes(command)) {
+                  const [x2Coord, y2Coord, xCoord, yCoord] = rest;
+                  readableSegment += `, ${x2Coord} ${y2Coord}`;
+                  if (yCoord !== undefined) {
+                    readableSegment += `, ${xCoord} ${yCoord}`;
+                  }
+                  if (command === lowerShorthandCurveCommand && (x2Coord !== 0 || y2Coord !== 0)) {
+                    readableSegment += ` (should be "l${x2Coord} ${y2Coord}" or removed)`;
+                  }
+                  if (command === upperShorthandCurveCommand) {
+                    readableSegment += ` (should be "L${x2Coord} ${y2Coord}" or removed)`;
+                  }
+                  if (command === lowerCurveCommand && (xCoord !== 0 || yCoord !== 0)) {
+                    readableSegment += ` (should be "l${xCoord} ${yCoord}" or removed)`;
+                  }
+                  if (command === upperCurveCommand) {
+                    readableSegment += ` (should be "L${xCoord} ${yCoord}" or removed)`;
+                  }
+                }
+                reporter.error(`Unexpected segment ${readableSegment} in path.`);
```

Comment:
Another minor suggestion. "Unexpected" seems a bit ambiguous... If we use "Ineffective" to maintain the consistence?

```suggestion
                reporter.error(`Ineffective segment ${readableSegment} in path.`);
```

Reply:
Thanks again for the review. I applied your suggestions!